python/S3_BUCKET_ENCRYPTED_KMS/S3_BUCKET_ENCRYPTED_KMS.py

```diff
@@ -0,0 +1,83 @@
+'''
+#####################################
+##           Gherkin               ##
+#####################################
+Rule Name:
+S3_BUCKET_ENCRYPTED_KMS
+
+Description:
+Check whether Amazon S3 bucket is encrypted with KMS key.
+
+Rationale:
+Ensure that the Amazon S3 bucket is always encrypted with the KMS key so that no clear text documents.
+
+Indicative Severity:
+High
+
+Trigger:
+Periodic
+
+Reports on:
+AWS::S3::Bucket
+
+Rule Parameters:
+None
+
+Scenarios:
+Scenario: 1
+  Given: There is no S3 Bucket
+   Then: Return NOT_APPLICABLE
+Scenario: 2
+  Given: S3 Bucket is encrypted with AWS-KMS
+   Then: Return COMPLIANT
+Scenario: 3
+  Given: S3 Bucket is not encrypted
+   Then: Return NON_COMPLIANT
+Scenario: 4
```

Comment:
merge 3 and 4. "S3 bucket is not encrypted with "KMS". By the way, we are checking for the "default encryption". https://docs.aws.amazon.com/AmazonS3/latest/dev/bucket-encryption.html

We need to rephrase the rule to be about S3_DEFAULT_ENCRYPTION_KMS